woodwork/utils.py

```diff
@@ -515,6 +515,20 @@ def concat_columns(objs, validate_schema=True):
 
     combined_df = lib.concat(objs, axis=1, join="outer")
 
+    # The lib.concat breaks the woodwork schema for dataframes with different shapes
+    # or mismatched indices.
+    null_cols = combined_df.isnull().any()[combined_df.isnull().any()].index
```

Comment:
Can you do something like `combined_df.columns[combined_df.isnull().any()]` here instead of using duplicate `.isnull().any()` calls?

Reply:
Or save the result of the calls into a variable so you don't have to repeat the operation.

```
mask = combined_df.isnull().any()
null_cols = mask[mask].index
```

Reply:
As you command